internal/k8s/controller.go

```diff
@@ -915,6 +915,11 @@ func (c *AviController) SetupEventHandlers(k8sinfo K8sinformers) {
 			namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(svc))
 			isSvcLb := isServiceLBType(svc)
 			var key string
+			if !lib.ValidServiceType(svc) {
```

Comment:
@ashishnayak96 We can move this logic above line number 916 (isServiceLBType) so that extra processing will be avoided else we can change if condition as `!isSvcLb && !lib.ValidServiceType`

Reply:
Will take in next iteration